### PR TITLE
Fix 21241 - Hide-Court-Order - Fixed-Review-Confirmation

### PR DESCRIPTION
**Pull request by ArwenQin**
*Issue #:* /bcgov/entity#Fix 21241

*Description of changes:*
- Fixed the issue in Review and Confirmation Page

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-694-bet6zmye.web.app

**Comment by ArwenQin** [comment]
Company for test: https://dev.business.bcregistry.gov.bc.ca/TUo32AUuGy?accountid=3302

File: src/views/Incorporation/IncorporationReviewConfirm.vue [hunk 1]
```diff
@@ -314,6 +314,15 @@ export default class IncorporationReviewConfirm extends Vue {
...
   }
+
+  /** Is true when the role is user, set the court order component to valid */
+  get isRoleUser (): boolean {
+    if (!this.isRoleStaff) {
+      this.setCourtOrderValidity(true)
+      return true
+    }
+    return false
+  }
```
Comment (severinbeauvais) [inline src/views/Incorporation/IncorporationReviewConfirm.vue:325]: I'm sure this works but there's a better, common way.

See https://github.com/bcgov/business-create-ui/blob/4d4a70be574b8cca57a76fe3a2aadac5466e02b3/src/store/store.ts#L746 and line 730.

Comment (ArwenQin) [reply src/views/Incorporation/IncorporationReviewConfirm.vue]: Thank you. I just updated store.js to fix this issue.

**Review by severinbeauvais — COMMENTED** [review]
LGTM.

Did you file an IA, while logged in as a regular user, to verify this?

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**